src/appComponents/twitterwall/singletweet.js

```diff
@@ -2,7 +2,7 @@
 angular
     .module('oe.twitterwall')
     .directive('tweet',tweet);
-/*directive to display tweets after a timespan of 10000 ms.*/
+/*directive to display tweets after a timespan of 8000 ms.*/
 tweet.$inject=['$timeout','$rootScope'];
 function tweet($timeout,$rootScope) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['tweet' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2414597727/issues/source?bid=3233857&fileBranchId=3235518#l7)